CHANGES.md

```diff
@@ -1,3 +1,20 @@
+## edge-20.1.3
+
+* CLI
+  * Introduced `linkerd check --cni` to ensure the CNI plugins can properly be
```

Comment:
This check is used to determine the status of the CNI after its installation. IOW, not whether the cluster meets all the preconditions of the CNI installation, but more about if the CNI installation completed successfully.

Reply:
Ah got it thanks!